### PR TITLE
Fixes #1631 : Fixed rare bug which broke menu bar

### LINKED ISSUE #1631: Unknown bug causes menu bar to disappear
**Actual Behaviour**

**Unknown Bug** causes menu bar to disappear.

**Expected Behaviour**

Should be bug-free.

**Steps to reproduce it**

openweb-app-gen-bug

**Log for the issue**

```
<div class="custom-menu-cont">
    <div class="custom-menu">
      <div class="hide-arrow"></div>
      <div class="custom-menu-cont">
        <div class="custom-menu-item">
          <a class="custom-icon" href="http://github.com/fossasia/open-event-webapp" target="_blank"><img src="./icons/code.png">
          <p class="custom-title">Code</p></a>
        </div>
        <div class="custom-menu-item">
          <a class="custom-icon" href="http://github.com/fossasia/open-event-webapp/issues" target="_blank"><img src="./icons/bug.png">
          <p class="custom-title">Bug Report</p></a>
        </div>
        <div class="custom-menu-item">
          <a class="custom-icon" href="http://blog.fossasia.org" target="_blank"><img src="./icons/blog.png">
          <p class="custom-title">Blogs</p></a>
        </div>
        <hr style="margin-bottom: 0; margin-top: 10px;">
        <div class="custom-menu-item">
          <a class="custom-icon" href="https://eventyay.com/" target="_blank"><img src="./icons/eventyay.png">
          <p class="custom-title">Eventyay</p></a>
        </div>
        <div class="custom-menu-item">
          <a class="custom-icon" href="https://susper.com/" target="_blank"><img src="./icons/susper.png">
          <p class="custom-title">Susper</p></a>
        </div>
        <div class="custom-menu-item">
          <a class="custom-icon" href="https://chat.susi.ai/" target="_blank"><img src="./icons/susi.png">
          <p class="custom-title">Susi</p></a>
        </div>
        <div class="custom-menu-item">
          <a class="custom-icon" href="https://fossasia.org/" target="_blank"><img src="./icons/fossasia.png">
          <p class="custom-title">FOSSASIA</p></a>
        </div>
        <div class="custom-menu-item">
          <a class="custom-icon" href="https://loklak.org/" target="_blank"><img src="./icons/loklak.png">
          <p class="custom-title">loklak</p></a>
        </div>
      </div>
    </div>
    <div class="custom-arrow"></div>
  </div>
```

**Would you like to work on the issue?**

Yes! *(Please don't send a PR without asking)*

### PR DESCRIPTION
**Pull request by 0x48piraj**
# Fixes #1631 

## Changes:
Applied two fixes.
* Removed the `custom-menu-cont` class from `child div`
* Added a `OnLoad()` function which focuses to the email.

## Demo for the change: 
openweb-app-gen-bug___fixed

### CODE REVIEW

**Comment by 0x48piraj** [comment]
Ping @geekyd @ParthS007 

Seems like Codacy is not analysing this `repo`.
image

**Comment by 0x48piraj** [comment]
Ping @geekyd

File: src/www/index.html [hunk 1]
```diff
@@ -156,7 +156,7 @@
   </style>
 </head>
 
-<body>
+<body onLoad="document.getElementById('email').focus();">
```
Comment (geekyd) [inline src/www/index.html]: Add this to `form.js`
https://github.com/fossasia/open-event-webapp/blob/development/src/www/js/form.js#L32

Comment (0x48piraj) [reply src/www/index.html]: Thanks @geekyd 
Done :+1:

**Pull request merged**